### PR TITLE
Argumens as objects for raw query

### PR DESCRIPTION
**Pull request by nikitin-da**

### CODE REVIEW

File: storio-common/src/test/java/com/pushtorefresh/storio/internal/InternalQueriesTest.java [hunk 1]
```diff
@@ -221,6 +260,39 @@ public void nonEmptyListOfStringsToNullableArrayOfStrings() {
...
+        List<Object> list = new ArrayList<Object>(3);
+        list.add(1);
+        list.add(null);
+        list.add(3);
+        String[] array = InternalQueries.nullableArrayOfStrings(list);
+
+        assertThat(array).isNotNull();
+
+        //noinspection ConstantConditions
+        assertThat(array.length).isEqualTo(list.size());
```
Comment (karlicoss) [inline storio-common/src/test/java/com/pushtorefresh/storio/internal/InternalQueriesTest.java]: you can use `assertThat(array).hasSameSizeAs`

File: storio-common/src/test/java/com/pushtorefresh/storio/internal/InternalQueriesTest.java [hunk 2]
```diff
@@ -99,7 +99,46 @@ public void nonEmptyListWithNullElementToUnmodifiableNonNullListOfStrings() {
...
     //endregion
 
-    //region Tests for Queries.unmodifiableNonNullList()
+    //region Tests for Queries.unmodifiableNonNullList() from array of objects
+
+    @SuppressWarnings("ConstantConditions")
+    @Test
+    public void nullArrayToUnmodifiableNonNullList() {
+        Object[] array = null;
+        assertThat(InternalQueries.unmodifiableNonNullList(array)).isEqualTo(emptyList());
```
Comment (karlicoss) [inline storio-common/src/test/java/com/pushtorefresh/storio/internal/InternalQueriesTest.java]: `isEqualTo` -> `isEmpty`?

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operations/put/DefaultPutResolver.java [hunk 3]
```diff
@@ -58,7 +58,7 @@ public PutResult performPut(@NonNull StorIOContentResolver storIOContentResolver
         final Query query = Query.builder()
                 .uri(updateQuery.uri())
                 .where(nullableString(updateQuery.where()))
-                .whereArgs((Object[]) nullableArrayOfStrings(updateQuery.whereArgs()))
+                .whereArgs((Object[]) nullableArrayOfStringsFromListOfStrings(updateQuery.whereArgs()))
```
Comment (karlicoss) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operations/put/DefaultPutResolver.java]: So, we still got this problem for update/delete queries right?

**Comment by karlicoss** [comment]
Ok, looks good except for minor comments on assertions.

File: storio-common/src/test/java/com/pushtorefresh/storio/internal/InternalQueriesTest.java (hunk 2 above)
Comment (nikitin-da) [inline storio-common/src/test/java/com/pushtorefresh/storio/internal/InternalQueriesTest.java]: I replaced it with `isSameAs` to check not just size, but also that it is unmodifiable

**Pull request merged**